app/src/main/java/com/markkolenbrander/capstonenewsapp/DetailFragment.kt

```diff
@@ -0,0 +1,38 @@
+package com.markkolenbrander.capstonenewsapp
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.markkolenbrander.capstonenewsapp.databinding.FragmentDetailBinding
+
+
+class DetailFragment : Fragment() {
+
+    private lateinit var binding: FragmentDetailBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+//        arguments?.let {
+//            articleList = it.getParcelable(ARG_LIST)!!
+//        }
+
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        binding = FragmentDetailBinding.inflate(inflater, container, false)
+
+        return binding.root
+    }
```

Comment:
You could enhance what you started in the onCreate() method and add it in here:

```suggestion
    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        binding = FragmentDetailBinding.inflate(inflater, container, false)
        
        (arguments.getParcelable(ARG_ARTICLE) as? Article)?.let { article ->
            binding.tvArticleAuthor.text = article.author
            ....
        }

        return binding.root
    }
```